src/crate/client/sqlalchemy/dialect.py

```diff
@@ -209,10 +210,13 @@ def get_schema_names(self, connection, **kw):
 
     @reflection.cache
     def get_table_names(self, connection, schema=None, **kw):
+        table_filter = "AND table_type = 'BASE TABLE'" \
+            if self.server_version_info >= TABLE_TYPE_MIN_VERSION else ""
         cursor = connection.execute(
-            "select table_name from information_schema.tables "
-            "where {0} = ? "
-            "order by table_name asc, {0} asc".format(self.schema_column),
+            "SELECT table_name FROM information_schema.tables "
```

Comment:
is this covered by tests?

Reply:
It wasn't. I pushed a fixup with a test.